src/Icons/doc/index.rst

```diff
@@ -43,6 +77,61 @@ define the HTML attributes added to the ``<svg>`` element:
     {{ ux_icon('user-profile', {height: '16px', width: '16px', aria-hidden: true}) }}
     {# renders <svg height="16" width="16" aria-hidden="true"> ... </svg> #}
 
+.. code-block:: yaml
+
+    # config/packages/ux_icons.yaml
+    ux_icons:
+        iconify:
+            enabled: true
+            on_demand: true
+            endpoint: 'https://api.iconify.design'
+
+Icon Sets
+~~~~~~~~~
+
+Icon Sets many icon sets available, each with their own unique style and set of icons
```

Comment:
`Icon Sets many icon sets available ...` -> `There are many icon sets available ...` ?

Reply:
😮‍💨  thank you